.eslintrc.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  root: true,
+  parser: "@typescript-eslint/parser",
+  plugins: ["@typescript-eslint"],
+  extends: [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "prettier",
+  ],
+  ignorePatterns: ["out/*", "*.js"],
+  overrides: [
+    {
+      files: ["*.ts"],
+      rules: {
+        "@typescript-eslint/no-explicit-any": "off",
+        "no-useless-escape": "off",
+        "no-case-declarations": "off",
+        "guard-for-in": "error",
+        "no-var": "error",
+        "curly": "error",
+        // quotes: ["error", "single"]
```

Comment:
The question is `single`, `double` or `backtick`?

Reply:
@gabro what do you think?

Reply:
I don't think this must be managed by eslint at all. For all formatting rules let's use prettier